### PR TITLE
Reorganizes directory structure

### PR DESCRIPTION
**Pull request by ashfurrow**
I should have done this from the get-go, but I didn't. This PR moves the danger files for issues and PRs into a directory instead of in the root of the repo. It also adds links to the different PR/issue events. 

@freak4pc going to assign this to you, but please let me merge the PR so I can restart the Peril server immediately afterward.

### CODE REVIEW

**Review by freak4pc — APPROVED** [review]
Fire at will :shipit:

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # peril
 
-Settings for the RxSwiftCommunity organization's [Peril](https://github.com/danger/peril) server. The server receives webhook events from every org repository (new issues and pull requests). We can hook into that in the `issue.ts` and `pr.ts` files, respectively. Usually we only use `pr.ts`.
+Settings for the RxSwiftCommunity organization's [Peril](https://github.com/danger/peril) server. The server receives webhook events from every org repository (new issues and pull requests). We can hook into that in the `org/all-issues.ts` and `org/all-prs.ts` files, respectively. Usually we only use `all-pr.ts`, but Peril affords us a lot of power and flexibility to respond to [pull request actions](https://developer.github.com/v3/activity/events/types/#pullrequestevent) and [issue actions](https://developer.github.com/v3/activity/events/types/#issuesevent).
```
Comment (freak4pc) [inline README.md]: One thing I noticed, you wrote `all-pr` and `all-prs` (plural)
Which is right?

Comment (freak4pc) [reply README.md]: @ashfurrow one minor comment

Comment (ashfurrow) [reply README.md]: The plural is what I meant, good catch.

**Comment by ashfurrow** [comment]
Lol, closing the PR triggered Peril before I could restart the server and it left a helpful comment letting us know it couldn't find the (renamed) file. 

EDIT: I redelivered the webhook event to Peril and it deleted the comment (indicating everything is working again). Here's what it looks like if anyone is interested:

screen shot 2018-01-02 at 1 27 10 pm

**Comment by freak4pc** [comment]
<p align="center"></p>

**Comment by orta** [comment]
It was so hard to debug things before I added that error

**Comment by freak4pc** [comment]
@orta seems super useful :)

**Pull request merged**